youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -44,6 +46,8 @@ def check_version(self):
                 self.basename, self.basename, required_version)
             if self._downloader:
                 self._downloader.report_warning(warning)
```

Comment:
Ok, but we should use a different message, because
> `Your copy of avconv is outdated, update avconv to version 10-0 or newer if you encounter any errors.`

doesn't tell the user that it can't be used for `bestvideo+bestaudio` and that it won't get the best quality.